pages/shapeshifting-ritual-objects.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <loading-message v-if="loading" />
+  <map-component
+    v-else
+    :pageInfo="pageInfo"
+    :originalMarkers="originalMarkers"
+    :filtersGeneralInfo="filtersGeneralInfo"
+    :filterProperties="filterProperties"
+    :includeTimeline="false"
+    iconBehaviour="constant"
+  >
+  </map-component>
+</template>
+
+<script>
```

Comment:
I'm not sure the filtering is working on this page. If I unselect all ritual objects and shapeshifting categories, there are still witches on the map even though at the top of the drawer is says 0 witches displayed. Are you able to check this please Ruby? Thank you!

Reply:
have just realised the filtering doesnt work on any of the pages when clustering is on cause the markers only renders at the start, it works on normal map, am working on fixing this now but am struggling

Reply:
have managed to find a solution where the markers array is refilled whenever mapMarkers is updated and the the map is refreshed by removing markers and adding updated markers. i don't think this is an ideal solution as it removes then readds markers but can't think of anything else. let me know if you have better solutions!

Reply:
because I can't see any obvious issues performance wise, and because we are under time pressure, I think we leave this as is. Thank you for fixing!